utils/spacewalk-common-channels.ini

```diff
@@ -2735,6 +2735,53 @@ checksum = sha256
 base_channels = debian-10-pool-amd64-uyuni
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/Debian10-Uyuni-Client-Tools/Debian_10/
 
+[debian-11-pool-amd64-uyuni]
+label    = debian-11-pool-amd64-uyuni
+checksum = sha256
+archs    = amd64-deb
+repo_type = deb
+name     = Debian 11 (bullseye) pool for amd64 for Uyuni
+gpgkey_url =
+gpgkey_id =
+gpgkey_fingerprint =
+repo_url = http://deb.debian.org/debian/dists/bullseye/main/binary-amd64/
+
+[debian-11-amd64-main-updates-uyuni]
+label    = debian-11-amd64-main-updates-uyuni
+name     = Debian 11 (bullseye) AMD64 Main Updates for Uyuni
+archs    = amd64-deb
+repo_type = deb
+checksum = sha256
+base_channels = debian-11-pool-amd64-uyuni
+repo_url = http://deb.debian.org/debian/dists/bullseye-updates/main/binary-amd64/
+
+[debian-11-amd64-main-security-uyuni]
+label    = debian-11-amd64-main-security-uyuni
+name     = Debian 11 (bullseye) AMD64 Main Security for Uyuni
+archs    = amd64-deb
+repo_type = deb
+checksum = sha256
+base_channels = debian-11-pool-amd64-uyuni
+repo_url = http://security-cdn.debian.org/debian-security/dists/bullseye-security/updates/main/binary-amd64/
+
+[debian-11-amd64-uyuni-client-devel]
+label    = debian-11-amd64-uyuni-client-devel
+name     = Uyuni Client Tools for Debian 10 Buster AMD64 (Development)
+archs    = amd64-deb
+repo_type = deb
+checksum = sha256
+base_channels = debian-11-pool-amd64-uyuni
+repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/Debian11-Uyuni-Client-Tools/Debian_11/
+
+[debian-11-amd64-uyuni-client]
+label    = debian-11-amd64-uyuni-client
+name     = Uyuni Client Tools for Debian 10 Buster AMD64
+archs    = amd64-deb
+repo_type = deb
+checksum = sha256
+base_channels = debian-11-pool-amd64-uyuni
+repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/Debian11-Uyuni-Client-Tools/Debian_11/
```

Comment:
@juliogonzalez :
Directory https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/Debian11-Uyuni-Client-Tools/Debian_11/ is missing

Reply:
That's expected. It won't exist until we release for the first time.